Cargo.toml

```
@@ -17,7 +17,7 @@ documentation = "https://docs.rs/display_json"
 proc-macro = true
 
 [dependencies]
-syn = { version = "^1.0.0", features = ["derive", "printing", "extra-traits", "parsing"] }
+syn = { version = "^2.0", features = ["derive", "printing", "extra-traits", "parsing"] }
```

While we are at it, let's remove the superfluous caret as well :slightly_smiling_face::

```suggestion
syn = { version = "2", features = ["derive", "printing", "extra-traits", "parsing"] }
```

---

done